Example/iOS Example/AppDelegate.swift

```diff
@@ -43,6 +43,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 return
         }
         PiwikTracker.configureSharedInstance(withSiteID: siteid, baseURL: baseurl)
+        PiwikTracker.shared?.logger?.minLevel = .info
```

Comment:
I agree with @brototyp , I also think that users would like to have a shared instance to as a convenient way to use a logger. More ambitious users can use their own instances of loggers.